toast-tk-runtime/src/main/java/io/toast/tk/runtime/parse/FileHelper.java

```diff
@@ -44,12 +52,115 @@ private FileHelper() {
 	public static InputStream getInputStream(final String filename) {
 		LOG.debug("Open input stream: " + filename);
 		String fullFileName = filename;
-		if (!filename.startsWith("\\")) {
+		if (!(filename.startsWith("\\") || filename.startsWith("C:"))) {
 			fullFileName = "/" + fullFileName;
 		}
 		return FileHelper.class.getResourceAsStream(fullFileName);
 	}
 
+	public static String readFile(String fileName) {
+		String lines = "";
+		try {
+			lines = new String(Files.readAllBytes(Paths.get(fileName)), StandardCharsets.UTF_8);
+			return lines;
+		} 
+        catch(IOException ex) {
+        	String exception = "Error reading file '" + fileName + "' caused by " + ex.getMessage();
+        	LOG.info(exception);
+            return exception;       
+        }
+    }
+
+	public static String getStringFromInputStream(InputStream is) {
+		BufferedReader br = null;
+		StringBuilder sb = new StringBuilder();
+		
+		String line;
+		try {
+			br = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8));
+			while ((line = br.readLine()) != null) {
+				sb.append(line).append(System.lineSeparator());
+			}
+		} catch (IOException e) {
+			e.printStackTrace();
+		} finally {
+			if (br != null) {
+				try {
+					br.close();
+				} catch (IOException e) {
+					e.printStackTrace();
+				}
+			}
+		}
+		return sb.toString();
+	}
+	
+	public static void createDirectory(String arg) {
+		File theDir = new File(arg);
+
+		if (!theDir.exists()) {
+			System.out.println("creating directory: " + theDir.getName());
+	
+		    try{
+		        theDir.mkdir();
+		    } 
+		    catch(SecurityException se){
+		        throw se;
+		    }        
+		    
+		    System.out.println("DIR created"); 
+		}
+	}
+
+	public static void writeFile(String arg, String fileName) throws IOException {
+		 try {
+			 FileOutputStream fileWriter =   new FileOutputStream(fileName);
+	            BufferedWriter bufferedWriter = new BufferedWriter(
+	            		new OutputStreamWriter(fileWriter, StandardCharsets.UTF_8));
+	            bufferedWriter.write(arg);
+	            bufferedWriter.close();
+	        }
+	        catch(IOException ex) {
+	            throw new IOException("Error writing to file '" + fileName + "' cause by " + ex.getMessage());
+	        }
+	}
+
+	public static String getLastModifiedFile(String directoryPath) {
+		File directory = new File(directoryPath);
+		File moreRecentFile = null;
+		
+		if (!directory.exists()) {
+			LOG.warn("The file/directory '" + directoryPath + "' does not exist");
+		} else if (!directory.isDirectory()) {
+			LOG.warn("The path '" + directoryPath + "' does not match to a directory");
+		} else {
+			File[] subfiles = directory.listFiles();
+			
+			if( subfiles.length > 0 ){
+				moreRecentFile = subfiles[0];
+				for (int i = 1; i < subfiles.length; i++) {
+					File subfile = subfiles[i];
+					//System.out.println(subfile.getName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)